### DISCUSSION TITLE
Contributor Call: 18 March 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3637] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 18 March 2023 **5pm** UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/mar-18-2023/5pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.
⚠Watch out for timezone changes!

### Agenda

**Comment by tangerineshark** [comment, marked as answer]
**Contributor Call Notes**
Hey hey! @tangerineshark took these notes, so they're not that good! Change 'em up with abandon!

Participants:

@tangerineshark
@lexander2002
@joostdecock
@bobgeorgethe3rd
@BenJamesBen
@woutervdub
@darigovresearch
@nchilada
@starfetch 
H0rt3ns14
...am I forgetting anyone?

Skully - @woutervdub has been working on a pattern for a skull. Should it be the hallmark pattern for V3? Consensus is yes.

@eriese merged the cutting layout stuff, so you can play with it! Check it out! @BenJamesBen has been tinkering and troubleshooting.

@joostdecock is working on a feature where you could split your config and pattern display into separate browser windows, letting you have (e.g.) all the options open in one window, and the pattern display in another
- @woutervdub would like a button to flip the view to landcape
- @BenJamesBen is in favor
- @nchilada says options would be cool in an I-frame, and @joostdecock confirms this is already done in V3
- @starfetch votes for more than one column of options, and a small discussion is had about opportunities for layout that multiple browser windows would enable

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
